examples/dmd/nonlinear_elasticity.cpp

```diff
@@ -430,10 +434,17 @@ int main(int argc, char *argv[])
     dmd_training_timer.Start();
 
     // 10. Create DMD object and take initial sample.
-    CAROM::DMD dmd_x(x_gf.GetTrueVector().Size(), dt);
-    CAROM::DMD dmd_v(v_gf.GetTrueVector().Size(), dt);
-    dmd_x.takeSample(x_gf.GetTrueVector(), t);
-    dmd_v.takeSample(v_gf.GetTrueVector(), t);
+    int curr_window = 0;
+    vector<CAROM::DMD*> dmd_x;
+    vector<CAROM::DMD*> dmd_v;
+    vector<double> tw;
```

Comment:
What does tw do? It does not look like it is used.

Reply:
Ah, perhaps it is the same as ts. We already have ts, so I think tw can be removed.